float32 x # coord x position
float32 y # coord y position

float32 vel_x # velocity on coord x
float32 vel_y # velocity on coord y
